webpack/renderer.ts

```diff
@@ -85,6 +87,10 @@ export function webpackLensRenderer({ showVars = true } = {}): webpack.Configura
     },
 
     plugins: [
+      new DefinePlugin({
+        CONTEXT_MATCHER_FOR_NON_FEATURES: `/\\.injectable(\\.${platform})?\\.tsx?$/`,
+        CONTEXT_MATCHER_FOR_FEATURES: `/\\/(renderer|common)\\/.+\\.injectable(\\.${platform})?\\.tsx?$/`,
+      }),
```

Comment:
So this means that we don't have to use `isMac`, `isWindows` etc. anymore? We can now introduce injection token and then specify the implementation in `something.injectable.mac.ts`, `something.injectable.windows.ts`

Reply:
> So this means that we don't have to use isMac, isWindows etc. anymore?

Yes we should be able to move towards that.

> We can now introduce injection token and then specify the implementation in something.injectable.mac.ts, something.injectable.windows.ts?

Correct. However, something about this PR breaks on windows CI (not locally on my windows machine though) so I need to figure out why.

Reply:
Ok, cool.